src/components/Input/Input.component.tsx

```diff
@@ -62,33 +69,36 @@ export const Input = forwardRef<HTMLInputElement, InputProps>((props, ref) => {
 		type: string
 	) => {
 		if (!readOnly) {
-			event.target.placeholder = (type === 'focus') ? '' : inputLabel;
+			event.target.placeholder = type === 'focus' ? '' : inputLabel;
 		}
 	};
 
 	return (
-		<StyledInputContainer
-			className={`mm-input ${className}`}
-			fullWidth={fullWidth}
-		>
-			{iconName && <Icon name={iconName} size={16}/>}
-			<StyledInput
-				ref={ref}
-				placeholder={inputLabel}
-				onFocus={(event: React.ChangeEvent<HTMLInputElement>) =>
-					togglePlaceholderValue(event, 'focus')
-				}
-				onBlur={(event: React.ChangeEvent<HTMLInputElement>) =>
-					togglePlaceholderValue(event, 'blur')
-				}
-				{...restProps}
-			/>
-			{searchQuery && (
-				<StyledIconButton onClick={onClose}>
-					<Icon name='Close' size={12} iconColor='#ffffff'/>
-				</StyledIconButton>
-			)}
-			<DisplayFieldSet value={value} label={inputLabel} error={error} />
-		</StyledInputContainer>
+		<div>
+			<StyledInputContainer
+				className={`mm-input ${className}`}
+				fullWidth={fullWidth}
+			>
+				{iconName && <Icon name={iconName} size={16} />}
+				<StyledInput
+					ref={ref}
+					placeholder={inputLabel}
+					onFocus={(event: React.ChangeEvent<HTMLInputElement>) =>
+						togglePlaceholderValue(event, 'focus')
+					}
+					onBlur={(event: React.ChangeEvent<HTMLInputElement>) =>
+						togglePlaceholderValue(event, 'blur')
+					}
+					{...restProps}
+				/>
+				{searchQuery && (
+					<StyledIconButton onClick={onClose}>
+						<Icon name="Close" size={12} iconColor="#ffffff" />
+					</StyledIconButton>
+				)}
+				<DisplayFieldSet value={value} label={inputLabel} error={error} />
+			</StyledInputContainer>
+			{Boolean(error) && <InputErrorMessage>{error}</InputErrorMessage>}
```

Comment:
```suggestion
			{error && <InputErrorMessage>{error}</InputErrorMessage>}
```
I think this should work

Reply:
Yes it will work, just for consistency i am using Boolean. At other places also we are first converting them to boolean